apps/snitch_core/lib/core/data/schema/country.ex

```diff
@@ -1,8 +1,11 @@
-defmodule Core.Snitch.Country do
-  use Ecto.Schema
-  import Ecto.Changeset
-  alias Core.Snitch.State
-  alias __MODULE__, as: Country
+defmodule Snitch.Data.Schema.Country do
+  @moduledoc """
+  Models a Country.
+  """
+
+  use Snitch.Data.Schema
+  alias Snitch.Data.Schema.State
+  alias __MODULE__
```

Comment:
No use of `Country`, so no need of this alias.

Reply:
it's used in the changeset.